### DISCUSSION TITLE
Database installation from scratch

### DISCUSSION THREAD

**Discussion by matamoros** [oschwengers/bakta#127] [Help wanted]
Hi,

I decided to give Bakta a try today, but I can't seem to install the database.

I installed Bakta via Conda without errors and the software seems to work.

However when I try to use bakta_db I get an error message saying: "Command not found". I feel like I am missing something very obvious, but I can't figure out what. I tried bakta_db list; bakta_db --help; bakta bakta_db list... What am I missing?

**Comment by oschwengers** [comment]
HI @matamoros ,
thanks for trying Bakta. To check, I just installed Bakta into a fresh conda environment and tested `bakta_db list` without any issues:
```bash
$ conda create -p test-conda-v150 bakta=1.5.0
....
$ conda activate ./test-conda-v150
$ bakta_db list
Bakta software version: 1.5.0
Required database schema version: 4

Available DB versions:
4.0	2022-08-29	10.5281/zenodo.7025248
$ bakta_db download -h
usage: bakta_db download [-h] [--output OUTPUT] [--minor MINOR]

options:
  -h, --help            show this help message and exit
  --output OUTPUT, -o OUTPUT
                        output directory (default = current working directory)
  --minor MINOR, -n MINOR
                        Database minor version (default = most recent db minor version)
```
Are your sure, that you've activated the right environment and that there are no version conflicts? Sometimes it's best to try with a fresh environment first.

**Reply by matamoros** [reply]
Thanks. It worked indeed with a new environment. There was a conflict with an older version of Python.
